nipype/interfaces/afni/tests/test_auto_ROIStats.py

```diff
@@ -25,9 +25,8 @@ def test_ROIStats_inputs():
     quiet=dict(argstr='-quiet',
     position=1,
     ),
-    terminal_output=dict(mandatory=True,
+    terminal_output=dict(deprecated='1.0.0',
```

Comment:
can we set version temporarily to `1.0.0` and make sure nothing breaks :) doesn't have to be a in a test. could be your local test.

Reply:
Running locally with version `1.0.1`. Will also run some examples.

Reply:
Confirmed it runs correctly.